tensorflow/core/profiler/convert/op_stats_to_overview_page.cc

```diff
@@ -286,15 +286,16 @@ std::string TfFunctionRecommendationHtml(const TfFunctionDb& tf_function_db) {
   // Sorts candidates in descending order of expensive_call_percent.
   absl::c_sort(candidates, cmp);
   std::string expensive_functions = "";
-  auto num_functions_shown = std::min(
+  int64 num_functions_shown = std::min(
       static_cast<decltype(candidates)::size_type>(3), candidates.size());
 
-  for (auto i = 0; i < num_functions_shown; i++) {
+  for (int64 i = 0; i < num_functions_shown; i++) {
     if (i > 0) absl::StrAppend(&expensive_functions, ", ");
     absl::StrAppend(&expensive_functions, "\"", candidates[i].function_name,
                     "\"");
   }
-  if (candidates.size() > num_functions_shown)
+  int candidates_size  = candidates.size();
+  if (candidates_size > num_functions_shown)
```

Comment:
We control the type of `num_functions_shown`, can we make it to be the same as `candidates.size()`? If not, `const` when extracting the variable

Reply:
Don't forget this one too